server/connectors/notion_connector/client.py

```
@@ -0,0 +1,1032 @@
+                lambda: self._request(
+                    "GET", f"/databases/{database_id}/views"
+                ),
+                "views",
+            )
+        )
+
+    def query_view(
+        self,
+        view_id: str,
+        *,
+        filter: Optional[Dict[str, Any]] = None,
```

Ruff (0.15.10)

[error] Function argument `filter` is shadowing a Python builtin

(A002)